### PR TITLE
Login styling

### PR DESCRIPTION
**Pull request by finn00dev**

### CODE REVIEW

File: www/html/ci/application/views/base.html [hunk 1]
```diff
@@ -4,72 +4,76 @@
...
-    <title>Ken Torii &copy;</title>
-    <meta content="width=device-width, initial-scale=1" name="viewport" />
-    <link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" />
-    <link href="{{ base_url }}font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
-    <link href="{{ base_url }}css/index2.css" rel="stylesheet" />
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
-    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
-    <script type="text/javascript" src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
-    <script type="text/javascript" src="{{ base_url }}js/webJS.js"></script>
+    {% block head %}{% endblock %}
```
Comment (ktorii) [inline www/html/ci/application/views/base.html]: Can you explain this change? I don't think I should see the header change in base template. That's the whole point of having a template file.

File: www/html/ci/application/views/base.html [hunk 2]
```diff
@@ -4,72 +4,76 @@
...
+      </ul>
+</ul>
                     </div>
                 </div>
             </nav>
-
         </div>
     </div>
-    {% block content %}{% endblock %}
+    {% block content1 %}{% endblock %}
```
Comment (ktorii) [inline www/html/ci/application/views/base.html]: why is this renamed to content1

File: www/html/ci/application/views/landing.php [hunk 3]
```diff
@@ -1,11 +1,86 @@
...
 {% endblock %}
+{% block head %}
+
+
+	<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+	<title>Ken Torii &copy;</title>
+	<meta content="width=device-width, initial-scale=1" name="viewport" />
+	<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" />
+	<link href="{{ base_url }}font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
+	<link href="{{ base_url }}css/index2.css" rel="stylesheet" />
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: this header should be in the template

File: www/html/ci/application/views/base.html [hunk 4]
```diff
@@ -4,72 +4,76 @@
...
+          {% if true == session %}
+    			<td><a type="button" data-toggle="modal" data-target="#myModal"><i class="fa fa-2x fa-unlock" onmouseover="lock(this)" onmouseout="unlock(this)" id="lock" aria-hidden="true"></i></a></td>
+          {% else %}
+          <td><a type="button" data-toggle="modal" data-target="#myModal"><i class="fa fa-2x fa-lock" onmouseover="unlock(this)" onmouseout="lock(this)" id="lock" aria-hidden="true"></i></a></td>
+          {% endif %}
+    		</tr></table>
+
+    	</footer>
+
+    {% block content2 %}{% endblock %}
```
Comment (ktorii) [inline www/html/ci/application/views/base.html]: There shouldn't be any content after the footer.

File: www/html/ci/application/views/landing.php [hunk 5]
```diff
@@ -1,11 +1,86 @@
...
+		        <input class="form-control" type="text" name="username" value="" size="50" /><br>
+
+		        <h5>Password</h5>
+		        <input class="form-control" type="password" name="password" value="" size="50" />
+
+		        <div><input class="btn" type="submit" value="Login" /></div>
+
+		    </form>
+
+		    <script type='text/javascript'>
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: javascript should be in seperate file. This is russell's function right. Where was it before. If youc ant' call it from there, put it in the webjs file

File: www/html/ci/application/views/landing.php [hunk 6]
```diff
@@ -101,132 +176,65 @@
...
-									</div>
-								</a>
-								<a class="portfolio-hover-link" data-toggle="modal" href="#portfolioModal4">
-									<div class="portfolio-hover-col-half portfolio-hover-info">
-										<center>
-											<i class="fa fa-info fa-4x"></i>
-										</center>
-									</div>
-								</a>
-							</div>
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: You're deleting somebody else's newly added code.

File: www/html/ci/application/views/landing.php [hunk 7]
```diff
@@ -101,132 +176,65 @@
...
-									</center>
-								</div>
-								<a class="portfolio-hover-link" data-toggle="modal" href="#portfolioModal1">
-									<div class="portfolio-hover-col-half portfolio-hover-info">
-										<center>
-											<i class="fa fa-info fa-4x"></i>
-										</center>
-									</div>
-								</a>
-							</div>
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: same here as above

File: www/html/ci/application/views/landing.php [hunk 8]
```diff
@@ -101,132 +176,65 @@
...
-									</center>
-								</div>
-								<a class="portfolio-hover-link" data-toggle="modal" href="#portfolioModal5">
-									<div class="portfolio-hover-col-half portfolio-hover-info">
-										<center>
-											<i class="fa fa-info fa-4x"></i>
-										</center>
-									</div>
-								</a>
-							</div>
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: same as above.

File: www/html/ci/application/views/landing.php [hunk 9]
```diff
@@ -101,132 +176,65 @@
...
-									</center>
-								</div>
-								<a class="portfolio-hover-link" data-toggle="modal" href="#portfolioModal6">
-									<div class="portfolio-hover-col-half portfolio-hover-info">
-										<center>
-											<i class="fa fa-info fa-4x"></i>
-										</center>
-									</div>
-								</a>
-							</div>
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: same as above.

File: www/html/css/index2.css [hunk 10]
```diff
@@ -1,3 +1,37 @@
...
+  text-align: center;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  height: 80px;
+  width: 100%;
+  overflow:hidden;
+  z-index: 9999;
+}
+
```
Comment (ktorii) [inline www/html/css/index2.css]: add new code at the bottom of the file please. the top has general container rules that are nice to have at the top

File: www/html/ci/application/views/base.html [hunk 11]
```diff
@@ -4,72 +4,76 @@
...
     </div>
-    {% block content %}{% endblock %}
+    {% block content1 %}{% endblock %}
+
+
+    	<footer class="container-fluid center-block">
+    		<table><tr>
+    			<td><h6 id="copyright">Copyright KenTorii.com</h6></td>
+          {% if true == session %}
+    			<td><a type="button" data-toggle="modal" data-target="#myModal"><i class="fa fa-2x fa-unlock" onmouseover="lock(this)" onmouseout="unlock(this)" id="lock" aria-hidden="true"></i></a></td>
```
Comment (ktorii) [inline www/html/ci/application/views/base.html]: If I'm logged in, and then I click on the icon, then it should log me out immediately. Does your code do this? If not, please add it.

File: www/html/ci/application/views/landing.php [hunk 12]
```diff
@@ -587,16 +595,24 @@
...
+
+	{% block content2 %}
+
+
+
 </body>
 
 <!--ADMIN PAGE-->
 <div class="container kt-tabs-component-content clean-container-theme strictly-no-padding" id="adminpage">
 	<button type="button" id='logout'>Logout</button>
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: Let's remove this button. I want the logout to be the icon in the footer only.

**Review by ktorii — CHANGES_REQUESTED** [review]
You're deleting a lot of new code that I wrote in the landing.php file. Be careful

**Comment by finn00dev** [comment]
hey, a lot of the things that were deleted/moved were do to my attempt to fix merge errors. I will make all the necessary fixes and resubmit.

File: www/html/ci/application/views/base.html [hunk 13]
```diff
@@ -70,6 +70,19 @@
     </div>
     {% block content %}{% endblock %}
 
+
+    	<footer class="container-fluid center-block">
+    		<table><tr>
+    			<td><h6 id="copyright">Copyright KenTorii.com</h6></td>
+          {% if true == session %}
+    			<td><a type="button"><i class="fa fa-2x fa-unlock logout" onmouseover="lock(this)" onmouseout="unlock(this)" id="lock" aria-hidden="true"></i></a></td>
```
Comment (ktorii) [inline www/html/ci/application/views/base.html]: Can you make the logout class an id instead here. Since we deleted the logout button Russell put, we only have one thing that has the logout capability. So we should make it an id.

File: www/html/js/webJS.js [hunk 14]
```diff
@@ -36,16 +39,18 @@ $(document).ready(function() {
     });
 
 
-    $('#logout').click(function(event) {
+    $('.logout').click(function(event) {
```
Comment (ktorii) [inline www/html/js/webJS.js]: Then remember to change this to id

**Review by ktorii — CHANGES_REQUESTED** [review]

**Review by ktorii — APPROVED** [review]
